### PR TITLE
Add random suffix to all tables in TestDistributedQueries

### PR DESCRIPTION
**Pull request by alexjo2144**
Many of the tests were doing this already, but this updates the few that were missed.

### CODE REVIEW

File: presto-testing/src/main/java/io/prestosql/testing/sql/TestTable.java [hunk 1]
```diff
@@ -66,6 +66,11 @@ public void close()
         sqlExecutor.execute("DROP TABLE " + name);
     }
 
+    public static String getTestTableName(String baseTableName)
```
Comment (kokosing) [inline presto-testing/src/main/java/io/prestosql/testing/sql/TestTable.java:69]: Actually, do we need to pass any argument here? Maybe we could have only `getUniqueTableName()` or just `getUniqueName()`?

File: presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java [hunk 2]
```diff
@@ -349,94 +354,101 @@ protected void assertCreateTableAsSelect(Session session, @Language("SQL") Strin
...
+        String tableName = getTestTableName("test_drop_column");
+        assertUpdate("CREATE TABLE " + tableName + " AS SELECT 123 x, 111 a", 1);
 
-        assertUpdate("ALTER TABLE test_drop_column DROP COLUMN x");
-        assertQueryFails("SELECT x FROM test_drop_column", ".* Column 'x' cannot be resolved");
+        assertUpdate("ALTER TABLE " + tableName + " DROP COLUMN x");
+        assertQueryFails("SELECT x FROM " + tableName, ".* Column 'x' cannot be resolved");
 
-        assertQueryFails("ALTER TABLE test_drop_column DROP COLUMN a", ".* Cannot drop the only column in a table");
+        assertQueryFails("ALTER TABLE " + tableName + " DROP COLUMN a", ".* Cannot drop the only column in a table");
```
Comment (kokosing) [inline presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java:426]: This test is missing `DROP`

File: presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java [hunk 3]
```diff
@@ -154,41 +155,45 @@ public void testResetSession()
     @Test
     public void testCreateTable()
     {
-        assertUpdate("CREATE TABLE test_create (a bigint, b double, c varchar)");
-        assertTrue(getQueryRunner().tableExists(getSession(), "test_create"));
-        assertTableColumnNames("test_create", "a", "b", "c");
+        String tableName = getTestTableName("test_create");
```
Comment (kokosing) [inline presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java:158]: It looks like only few tests left... so maybe let's continue the convention from other tests.

File: presto-testing/src/main/java/io/prestosql/testing/sql/TestTable.java (hunk 1 above)
Comment (findepi) [reply presto-testing/src/main/java/io/prestosql/testing/sql/TestTable.java:69]: > Maybe we could have only getUniqueTableName() or just getUniqueName()?

that won't work. table name prefix is very important. it's sometimes the only useuful information in an err msg 

but i think we don't need this method; the inline version with manual concatenation is rather readable

Comment (alexjo2144) [reply presto-testing/src/main/java/io/prestosql/testing/sql/TestTable.java:69]: I can switch them to match the existing tests. My only preference for this was the extra `_` between the prefix and the random suffix.

File: presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java (hunk 3 above)
Comment (alexjo2144) [reply presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java]: Ah, I thought I had gotten them all. I just added a few that I missed the first round, but let me know if I missed any more.

File: presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java (hunk 2 above)
Comment (alexjo2144) [reply presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java:425]: Added

File: presto-testing/src/main/java/io/prestosql/testing/sql/TestTable.java (hunk 1 above)
Comment (alexjo2144) [reply presto-testing/src/main/java/io/prestosql/testing/sql/TestTable.java]: Just removed that method, opting for the inline string concat like the existing tests.

File: presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java [hunk 4]
```diff
@@ -349,94 +353,101 @@ protected void assertCreateTableAsSelect(Session session, @Language("SQL") Strin
...
+        String renamedTable = "test_rename_new_" + randomTableSuffix();
+        assertUpdate("ALTER TABLE " + tableName + " RENAME TO " + renamedTable);
+        assertQuery("SELECT x FROM " + renamedTable, "VALUES 123");
 
-        assertUpdate("ALTER TABLE test_rename_new RENAME TO TEST_RENAME"); // 'TEST_RENAME' is upper-case, not delimited
+        String uppercaseName = "TEST_RENAME_" + randomTableSuffix();
+        assertUpdate("ALTER TABLE " + renamedTable + " RENAME TO " + uppercaseName); // 'TEST_RENAME' is upper-case, not delimited
         assertQuery(
-                "SELECT x FROM test_rename", // 'test_rename' is lower-case, not delimited
+                "SELECT x FROM " + uppercaseName.toLowerCase(ENGLISH), // 'test_rename' is lower-case, not delimited
```
Comment (findepi) [inline presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java:366]: update the comment

**Comment by findepi** [comment]
Merged, thanks!

**Pull request merged**